plugins/media/wowza/services/LiveConversionProfileService.php

```diff
@@ -102,21 +108,28 @@ public function serveAction($streamName, $hostname = null)
 		$transcode = $root->addChild('Transcode');
 		
 		$encodes = $transcode->addChild('Encodes');
+		$defaultFrameRate = null;
+
 		$groups = array();
 		foreach($liveParams as $liveParamsItem)
 		{
 			/* @var $liveParamsItem liveParams */
 			if(!$liveParamsItem->hasTag(assetParams::TAG_SOURCE) && in_array($liveParamsItem->getId(), $ignoreLiveParamsIds))
 				continue;
-				
+
+			if ($liveParamsItem->hasTag(assetParams::TAG_SOURCE)) {
+				if ($liveParamsItem->getFrameRate() && $liveParamsItem->getFrameRate() !== 0) {
```

Comment:
Shouldn't there be a minimum value for the "getFrameRate" what if I put in 1 is that valid?